elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "mercurymedia/elm-ag-grid",
     "summary": "AgGrid integration for Elm",
     "license": "MIT",
-    "version": "10.1.0",
```

Comment:
Would be good to always have a CHANGELOG entry whenever we bump the version. So it's always clear what the version changes, even if it's just new exported functions.

Otherwise LGTM

Reply:
Ahh yes... I will add the changelog entry

Reply:
:heavy_check_mark: